unstract/connectors/src/unstract/connectors/databases/exceptions.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, detail: Any, database: Any) -> None:
             f"Error creating/writing to {database}. Syntax incorrect. "
             f"Please check your table-name or schema. "
         )
-        super().__init__(detail=default_detail)
+        final_detail = f"{default_detail}Details: {detail}" if detail else default_detail
```

Comment:
How does this get rendered in the UI? Don't we need this space? 
```suggestion
        final_detail = f"{default_detail}\nDetails: {detail}" if detail else default_detail
```